src/lib/queue.js

```diff
@@ -38,7 +38,12 @@ export default class EventQueue {
     if (typeof callback === 'function' && callback(subscriber) === false) {
       return;
     }
-    subscriber.handleEvent(name, data);
+    // We should catch potential errors globally
+    try {
+      subscriber.handleEvent(name, data);
```

Comment:
If we don't catch an error, the queue breaks and some of the plugins don't get any further events.

Reply:
Yep, sounds reasonable.